### PR TITLE
Fix possible timeouts on devtools project generation tests

### PR DESCRIPTION
**Pull request by ia3andy**
- Change timeout to 2s
- Use only 4 threads to reduce and make the duration stabler
- Local tested duration is under 250ms

### CODE REVIEW

**Review by FroMage — APPROVED** [review]
LGTM

**Comment by gsmet** [comment]
The Windows failure is unrelated, merging.

File: devtools/common/src/test/java/io/quarkus/cli/commands/CreateProjectTest.java [hunk 1]
```diff
@@ -243,6 +246,7 @@ void createMultipleTimes() throws InterruptedException {
         }).collect(Collectors.toList());
         executorService.invokeAll(collect);
         latch.await();
+        System.out.println(watch.toString());
```
Comment (gsmet) [inline devtools/common/src/test/java/io/quarkus/cli/commands/CreateProjectTest.java]: Hmmm, what's the purpose of this?

Comment (ia3andy) [reply devtools/common/src/test/java/io/quarkus/cli/commands/CreateProjectTest.java]: It logs the test duration. I thought it was not a problem to leave it here. should I remove it?

Comment (gsmet) [reply devtools/common/src/test/java/io/quarkus/cli/commands/CreateProjectTest.java]: I would say that I wouldn't import a Google class just for that. So I would remove it for now and revisit if the changes you made are not stable enough.

**Comment by gsmet** [comment]
@ia3andy you broke the formatting again. Can you fix? Thanks!

**Pull request merged**